File: 백준/Bronze/2754. 학점계산/학점계산.py
score = input()
if score == 'F':
    print("0.0")
else:
    output = 69 - ord(score[0]) + 0.0
    if score[1] == "+":
        output += 0.3
    elif score[1] == "-":
        output -= 0.3
    
    print(output)

File: 백준/Bronze/2754. 학점계산/학점계산.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		String score = sc.next();
		sc.close();
		if (score.equals("F")) {
			System.out.println("0.0");
		} else {
			float output = 69 - (int) score.charAt(0);
			if (score.charAt(1) == '+')
				output += 0.3;
			else if (score.charAt(1) == '-')
				output -= 0.3;
			System.out.println(output);
		}
	}
}